sys/riscv/conf/std.CHERI

```diff
@@ -0,0 +1,6 @@
+makeoptions	CPUTYPE=cheri
+options 	CPU_CHERI
+options		COMPAT_FREEBSD64
```

Comment:
It looks like there's a whitespace issue here.  Maybe "<tab><tab>" instead of "<space><tab>"?

Reply:
Fixed, I had it wrong (device uses two tabs, options uses space+tab).